signac/contrib/project.py

```diff
@@ -799,7 +799,7 @@ def detect_schema(self, exclude_const=False, subset=None, index=None):
         return ProjectSchema.detect(statepoint_index)
 
     @deprecated(deprecated_in="1.3", removed_in="2.0", current_version=__version__,
-                details="Use find_jobs().ids instead.")
+                details="Use [job.id for job in project.find_jobs()] to return job ids instead.")
```

Comment:
I'm not sure this is the right snippet to include here, or at least not the only snippet. Users may not always want to get all job ids directly like this. I think it would make sense to recommend just switching to `find_jobs`, then add another sentence indicating that users can access the ids of these jobs to replicate the original behavior, at which point you could include this list comprehension.